### PR TITLE
Add react-native-idle-timer (iOS only) to MeetingView.js

### PR DESCRIPTION
**Pull request by kb**
If the MeetingView.js view is active, then disable the idle timer so the
screen stays active and users can actively use the app with little friction.

### CODE REVIEW

File: MeetingNameInputView.js [hunk 1]
```diff
@@ -41,6 +41,9 @@ export default class MeetingNameInputView extends Component {
     const dataSource = new ListView.DataSource({
       rowHasChanged: (row1, row2) => row1 !== row2,
     });
+
+    // TODO @bolton: There is likely a better way than this
+    const events = props.events == null ? [] : props.events;
```
Comment (kb) [inline MeetingNameInputView.js:46]: @samn didn't you have a better fix for this? Maybe its local and not committed?

File: MeetingView.js [hunk 2]
```diff
@@ -38,6 +39,14 @@ export default class MeetingView extends Component {
     };
   }
 
+  componentWillMount() {
+    IdleTimerManager.setIdleTimerDisabled(true);
```
Comment (kb) [inline MeetingView.js:43]: This call only handles iOS right now. I suppose we can make a platform specific view or we can extend this bridge to support android.

**Comment by kb** [comment]
[May have found a module](https://github.com/yadi-social/react-native-dimmer) that does this for iOS and Android. Looking into it now.

File: MeetingNameInputView.js (hunk 1 above)
Comment (samn) [inline MeetingNameInputView.js:46]: Is this variable used?

**Comment by kb** [comment]
Closing in favor of https://github.com/kb/talking-stick-client/pull/3

**Pull request closed without merging**